requirements.txt

```diff
@@ -1,4 +1,4 @@
-django>=1.11,<2.2
+django>=2.0,<2.3
```

Comment:
Hi,

Why are you dropping `1.11`?
For one, it supports python 3.7 and second changing the requirements is not a part of this task.